extension/content.js

```diff
@@ -282,11 +299,18 @@ $(document).on('click', event => {
 	$target.closest('.js-details-container').toggleClass('refined-github-minimized');
 });
 
+// Handle issue list ajax
+$(document).on('pjax:end', event => {
+	addYoursMenuItem();
```

Comment:
I agree. There's also a (minor) performance impact in that this function runs on every page, not just those that are issue/PR searches.